setup.py

```diff
@@ -281,6 +281,7 @@ def finalize_options(self):
         'scipy >= 0.7.0',
         'six >= 1.5.0',
         'smart_open >= 1.2.1',
+        'morfessor==2.0.2alpha4',
```

Comment:
It's dependency for `VarEmbed` wrapper. Maybe need to create several mode for install.